src/TrackArtist.cpp

```diff
@@ -2184,18 +2184,20 @@ void TrackArtist::DrawClipSpectrum(WaveTrackCache &waveTrackCache,
 
    // nearest frequency to each pixel row from number scale, for selecting
    // the desired fft bin(s) for display on that row
-   float bins[ hiddenMid.height + 1 ];
+   std::vector<float> bins;
```

Comment:
Might help to call reserve of the anticipated size before calling push_back in a loop.

Or, use alloca.